invoker.js

```diff
@@ -206,6 +206,20 @@ function handleDefaultInvoke(event) {
   if (event.defaultPrevented) return;
 
   switch (event.target.localName) {
+    case "input": {
+      switch (event.action) {
+        case "auto":
+        case "showpicker": {
+          event.target.showPicker();
+          break;
+        }
+        case "click": {
+          event.target.click();
+          break;
+        }
```

Comment:
I'm not convinced we actually want the click option in the defaults. I think it should be showPicker. And if they want to do click they should do it manually. It's mostly a deprecated functionality which `showPicker` should completely replace.

Reply:
I agree. Let's not have `click`